docker-machine-nfs.sh

```diff
@@ -418,8 +457,8 @@ configureBoot2Docker()
   local file="/var/lib/boot2docker/bootlocal.sh"
 
   docker-machine ssh $prop_machine_name \
-    "echo '$bootlocalsh' | sudo tee $file && sudo chmod +x $file && sync" > /dev/null
-
+    "echo '$bootlocalsh' | sudo tee $file && sudo chmod +x $file && sync" < /dev/null > /dev/null
```

Comment:
Can you elaborate this change too? You made that change in different places ... wondering what the reason is, since this part does work quite well ..

Reply:
`docker-machine` on windows waits for keyboard input. So this script hangs until you press Enter. `< /dev/null` prevents this behavioral.

I've tested it also on Mac and it works without any issue.